### ISSUE TITLE
Iframe src not updating dynamcially.

### ISSUE THREAD

**Issue by saranyavelpuri** [redxtech/vue-plyr#69]
1. Iframe src not updating dynamcially.only on reload of full page only its updating. Being a SPA application, I am not supposed to reload the whole page.
2. Deleting the particular video, still iframe src is not updating.

**Comment by redxtech** [comment]
Try using a `key` directive on the `<vue-plyr>` element. Switch that when you change the source of the iframe. That should reload the player while keeping the page intact.
My recommendation would be setting the key to the source of the iframe, but you could use anything that would be unique to each different source (the query string at the end maybe?).

Hope this helps.

**Comment by dees040** [comment]
This worked for me:
```js
document.getElementById('youtube-player').src += '';
```

**Comment by dappiu** [comment]
> Try using a `key` directive on the `<vue-plyr>` element. Switch that when you change the source of the iframe. That should reload the player while keeping the page intact.
> My recommendation would be setting the key to the source of the iframe, but you could use anything that would be unique to each different source (the query string at the end maybe?).
> 
> Hope this helps.

This works but the player instance seems to lost the custom options with which was initialized.
I have this code

```
<no-ssr placeholder="Video player is loading...">
  <vue-plyr
    :key="videoSrc"
    class="is-unselectable"
    :options="{
      muted: true
    }"
  >
    <div class="plyr__video-embed">
      <iframe
        :src="videoSrc"
        allowfullscreen
        allowtransparency
        allow="autoplay;fullscreen"
      />
    </div>
  </vue-plyr>
</no-ssr>
```

where `videoSrc` is a computed property that returns something like `https://player.vimeo.com/video/<videoID>` and that changes when the user mouseenter a particular region of space.

Now, the player correctly picks up the updated videoSrc when videoSrc changes, but while on the first loaded video the `muted: true` option is respected, on subsequent videos the volume is not muted.

Any idea?

**Comment by dappiu** [comment]
It seems that I solved the problem I was talking about before by doing this:
I had `autopause: false`, because I thought that as the player instance got replaced by Vue, there was no need of pausing the previous video. This could be a wrong assumption, using the autopause produces more predictable results now.
Moreover, I had the storage enabled and all the player instances was using the key 'plyr', so if I changed the volume on some other player in another page of the website, the change reflected also for the players that was intentionally instantiated with `muted: true`. I disabled the storage for the player instance that is supposed to play a muted and autoplay-enabled video background which must stay muted all the time.
